app/views/templates/ChangeSuccessView.scala.html

```diff
@@ -31,12 +31,36 @@ <h2 class="heading-medium">@messages("common.whatHappensNext")</h2>
 
 @if(user.isAgent) {
 
-  @model.agentEmail match {
-    case Some(email) => {<p>@messages("changeSuccess.helpOne.digital.agent", email)</p>}
-    case _ => {<p>@messages("changeSuccess.helpOne.paper.agent")</p>}
-  }
+  @if(appConfig.features.bulkPaperOffEnabled()) {
```

Comment:
Do you think it's worth changing this `if` statement up a little bit so that it reuses the `updateClient` template for that second paragraph? That already handles the whole "your client" thing when business name is empty, so you wouldn't have to extract the optional value, and we could update the messages there instead of adding new ones.

Reply:
updateClient uses slightly different message content that's only applicable with the feature switch off so I didn't want to change that existing code/content